lib/browser-interface-iframe.js

```diff
@@ -63,6 +63,22 @@ class BrowserInterfaceIframe extends BrowserInterface {
 		this.wrapperDiv.remove();
 	}
 
+	async checkUrlValidity( pageUrl ) {
+		const checkResult = {
```

Comment:
I read `check-result` as "perform a check on the result" in code. Perhaps we can just simplify and call it `const result`?